src/mbgl/renderer/painter.hpp

```diff
@@ -151,6 +153,8 @@ class Painter : private util::noncopyable {
     LineAtlas* lineAtlas = nullptr;
 
     FrameHistory frameHistory;
+    gl::ShaderParameters shaderParameters;
+    gl::ShaderParameters shaderParametersOverdraw;
```

Comment:
`ShaderParameters` are only used to initialize the shaders; there's no need to keep them around.